website/blog/2022-01-19-version-067.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Announcing React Native 0.67
+authors: [kelset, lunaleaps]
+tags: [announcement, release]
+---
+
+# Announcing 0.67
+
+Happy new year everyone! Today we are announcing the latest release of React Native, 0.67.0, along with some updates on the release process that we have been working on in the past few months.
+
+### Sections
+
+- [Highlights of 0.67](/blog/2022/01/19/version-067#highlights-of-067)
+- [Improvements to Release Process](/blog/2022/01/19/version-067#improvements-to-release-process)
+- [Interested in helping React Native stabilise new releases?](/blog/2022/01/19/version-067#interested-in-helping-react-native-stabilise-new-releases)
+
+<!--truncate-->
+
+## Highlights of 0.67
+
+As mentioned in Meta's [H2 2021 plans](https://reactnative.dev/blog/2021/08/19/h2-2021), React Native is attempting more frequent releases for a shorter turnaround time for new features and fixes (like the new architecture) to land in the community. Naturally, many releases will focus on fixes and improvements.
+
+Here are some notable changes coming in 0.67.0:
+
+- Lean-core removals: [DatePickerAndroid](https://github.com/facebook/react-native/commit/7a770526c626e6659a12939f8c61057a688aa623#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346)
+- Rename deprecated Keyboard.removeEventListener to Keyboard.removeListener. [RN: Rename `Keyboard.remove{Event =>}Listener`](https://github.com/facebook/react-native/commit/4ad177efc7ad6a39b37103978c68c9b4d2b09c98)
```

Comment:
Is this notable? I thought this commit was a fix of a fix in a bug that a deprecation had in a previous release. Worth following up with Tim?

Reply:
I don't have strong opinions, either way it's fine. I would not prevent the blogpost to be merged because of this, so if it's easier let's just remove it
```suggestion
```